src/gigs-board/components/layout/Page.jsx

```diff
@@ -44,12 +44,75 @@ function href(widgetName, linkProps) {
 }
 /* END_INCLUDE: "common.jsx" */
 
+const Banner = styled.div`
+   {
+    position: fixed;
+    left: calc(-50vw + 50%);
+    width: 100vw;
+    height: 100px;
+    background: #232323;
+    z-index: 2;
+    margin-top: -24px;
+
+    overflow: hidden;
+  }
+`;
+
+const Logo = styled.div`
+   {
+    position: fixed;
+    padding: 32px 0;
+    z-index: 4;
+    margin-top: -24px;
+
+    img {
+      height: 36px;
+    }
+  }
+`;
+
+const Scroll = styled.div`
+   {
+    z-index: -1;
+    margin-top: calc(-24px + 100px + 25px);
+  }
+`;
+
+const Gradient = styled.div`
+   {
+    position: absolute;
+    left: -48vw;
+    width: 173vw;
+    height: 70vw;
+    top: -40vw;
+    transform: rotate(9.78deg);
+    background: linear-gradient(
+      -90deg,
+      rgb(0, 204, 255) 0.57%,
+      rgb(50, 82, 166) 100.57%
+    );
+    opacity: 0.22;
+    filter: blur(17vw);
+    z-index: 3;
+  }
+`;
+
 return (
   <>
-    {widget("components.layout.Controls")}
-    {widget("components.layout.Navbar", {
-      children: props.navbarChildren,
-    })}
-    {props.children}
+    <Banner>
+      <Gradient></Gradient>
+    </Banner>
+    <Logo>
+      <a href="https://devgovgigs.near.social">
```

Comment:
This will take users from alternative NEWR Social gateways to near.social, and it also adds unnecessary redirects. Use `href` function pointing to the Feed page instead.